website/server/controllers/api-v3/tasks/groups.js

```diff
@@ -1,5 +1,5 @@
 import { authWithHeaders } from '../../../middlewares/auth';
-import { ensureDevelpmentMode } from '../../../middlewares/ensureDevelpmentMode';
+import ensureDevelpmentMode from '../../../middlewares/ensureDevelpmentMode';
```

Comment:
Ah, yea. Looks like it was merged in like that. That was my bad. I can fix that Ina separate PR. Other files will be affected.